content/applications/hr/recruitment/offer_job_positions.rst

```diff
@@ -0,0 +1,135 @@
+applicant's card.
+
+The next step is to send an offer to the applicant. On the applicant's card, click the
+:guilabel:`Generate Offer` button. A :guilabel:`Generate a Simulation Link` pop-up window appears.
+Most fields are pre-populated with information from the job position. If any necessary fields are
+blank, or if any information needs to be updated, enter or update the relevant information in the
+corresponding fields.
+
+.. note::
+   Depending on the localization setting for the company, and which applications are installed, some
+   fields may not appear in the :guilabel:`Generate a Simulation Link` pop-up window.
+
+   For example, if the *Fleet* application is **not** installed, any fields related to vehicles do
+   **not** appear.
+
+Universal fields
+----------------
+
+The following fields appear for all offers sent to applicants, regardless of the localization.
+
+- :guilabel:`Contract Template`: the template currently being used to populate the
+  :guilabel:`Generate a Simulation Link` pop-up window. Use the drop-down menu to select a different
+  :guilabel:`Contract Template`, if desired.
+
+  To modify the template, hover over the current template name, and click the
+  :icon:`oi-launch` :guilabel:`Internal link` icon that appears to the right of the drop-down arrow.
+  Make any desired changes, then click :guilabel:`Save & Close`.
+- :guilabel:`Job Position`: the name of the :guilabel:`Job Position` being offered to the applicant.
+  The selections available in the drop-down menu correspond to the :guilabel:`Job Positions`
+  configured on the main *Recruitment* dashboard.
+- :guilabel:`Job Title`: the specific name of the position being offered to the applicant.
+- :guilabel:`Department`: the :guilabel:`Department` the job position falls under.
+- :guilabel:`Contract Start Date`: the date the contract takes effect. The default date is the
+  current date. To modify the date, click on the displayed date to reveal a calendar pop-up window.
+  Navigate to the desired month, then click the day to select the date.
+- :guilabel:`Yearly Cost`: the annual salary being offered.
+- :guilabel:`Link Expiration Date`: the number of days the job offer is valid. The default
+  expiration date is `30` days. Modify the expiration date, if desired.
+
+Send offer
+----------
+
+When the :guilabel:`Generate a Simulation Link` pop-up window information is correct, click
+:guilabel:`Send By Email`, and an email pop-up window loads.
+
+If the applicant does not have an email address listed on their applicant card, a warning appears in
+a red box at the bottom of the :guilabel:`Generate a Simulation Link` pop-up window, stating:
+:guilabel:`The applicant does not have a valid email set. The Offer Link won't be able to be
+completed.` Click :guilabel:`Discard`, then enter an email on the applicant's card. Once an email is
+entered, click the :guilabel:`Generate Offer` button, and the email pop-up window loads again.
```

Comment:
can possibly put this in a warning admonition?
```suggestion
.. warning::
  The email pop-up fails to send if the applicant does not have an email address listed on their applicant card.
In the draft email pop-up window, click :guilabel:`Discard` to return to the applicant card. After entering the email, click the :guilabel:`Generate Offer` button to restart the process.
```

Reply:
I like the idea of putting it in an admonition- I put it in an 'important' since I feel a warning might be tot aggressive for this =D